cli/pyproject.toml

```diff
@@ -13,6 +13,7 @@ requests = "^2.28.2"
 click = "^8.1.3"
 pyyaml = "^6.0"
 scaleway = "^0.12.0"
+pyjwt = "^2.7.0"
```

Comment:
I think that can be a dev dependency since it's only used in the integrations test